### DISCUSSION TITLE
Errors on ./fresh-start.sh fails

### DISCUSSION THREAD

**Discussion by vranki** [evertramos/nginx-proxy-automation#371] [Q&A] [answered]
**Describe the bug**

Running the fresh-start.sh as described in README fails:

```bash
cd proxy/bin && ./fresh-start.sh --yes -e your_email@domain --skip-docker-image-check
⚡ Start execution '/home/cos/proxy/bin/fresh-start.sh  ✘ ERROR   You dont have permission to write log at '/var/log/basescript.log' - [log]
 ✘ ERROR   Logs could not be saved at '/var/log/basescript.log' - [log]
date: unrecognized option '--skip-docker-image-check'
Try 'date --help' for more information.
 ✘ ERROR   Logs could not be saved at '-e/your_email@domain' - [log]
 ✔ success Function 'check_local_env_file'
 ✔ success Function 'check_docker'
 ✔ success Function 'check_running_script'
 ✔ success Function 'starts_initial_check'
unknown flag: --file
See 'docker --help'.

Usage:  docker [OPTIONS] COMMAND

```
..and various other errors continue.

**Server info (please complete the following information):**
 - Linux release Ubuntu 22.04
 - Server type vps
 - Docker version Docker version 20.10.12, build 20.10.12-0ubuntu4
 - docker-compose version docker-compose version 1.29.2, build unknown

**Comment by evertramos** [comment, marked as answer]
Did you clone using submodules?

```bash
git clone --recurse-submodules https://github.com/evertramos/nginx-proxy-automation.git proxy 
```

**Reply by evertramos** [reply]
You could now run:

```bash
./bin/fix-git-submodule.sh
```

and try again!

**Reply by Realvincentyuan** [reply]
it does not work, same error `Logs could not be saved at ...`
Server info (please complete the following information):

Linux release Ubuntu 22.04
Server type vps
Docker version 24.0.4, build 3713ee1
Docker Compose version v2.19.1

**Reply by evertramos** [reply]
I have updated the logging system to local folder due to the diversity environment and permissions. The next version will be easier to configure the logging path, for now it will be in ./bin/basescript.log file. Please update to main and try again.

**Comment by gu00col** [comment]
@evertramos i clone using --recurse-submodules, i have erros too, in my case i do exacly tutorial and recivied error, 

unknown flag: --env-file

**Reply by evertramos** [reply]
you will need to send some more info and logs to try to help, probably you have not setup the .env file...

**Comment by gu00col** [comment]
Validating a previous version that I downloaded last year from the repository, it ran with the tutorial commands.
